CleanWeatherTests/Networking/FakeWeatherNetworking.swift

```diff
@@ -35,9 +35,9 @@ final class FakeWeatherNetworking: WeatherNetworking {
         fetchForecastWeatherForCityCompletion = completion
     }
 
-    func fetchHistoricalWeatherForCity(id: String, date: Double, completion: FetchHistoricalCompletion?) {
+    func fetchHistoricalWeatherForCity(coordinates: String, date: Double, completion: FetchHistoricalResponseCompletion?) {
```

Comment:
for all coordinates make a struct with lat long, in it from string and to string method